Sources/SwiftInspectorCore/InitializerAnalyzer.swift

```diff
@@ -180,3 +203,17 @@ private final class InitializerSyntaxReader: SyntaxVisitor {
   let shouldVisitIdentifier: (String) -> Bool
   let onNodeVisit: (InitializerDeclSyntax) -> Void
 }
+
+private final class FunctionParameterReader: SyntaxVisitor {
```

Comment:
I like the second syntax visitor approach

Reply:
Thanks! Yeah I think it makes it a lot cleaner to follow the code.